aptos/token_bridge/sources/transfer_tokens.move

```diff
@@ -62,6 +68,39 @@ module token_bridge::transfer_tokens {
         )
     }
 
+    public entry fun transfer_tokens_with_payload_with_signer<CoinType>(
+        sender: &signer,
+        amount: u64,
+        recipient_chain: u64,
+        recipient: vector<u8>,
+        wormhole_fee: u64,
+        nonce: u64,
+        payload: vector<u8>
+    ): u64 acquires EmitterCapabilityStore {
+        if (!exists<EmitterCapabilityStore>(signer::address_of(sender))) {
```

Comment:
it might be nicer if we lifted this out to a separate function like `get_emitter_cap_signer` (or something like that) that creates one if and moves it to the signer if it doesn't exist yet.